app/models/product.rb

```diff
@@ -0,0 +1,12 @@
+class Product < ApplicationRecord
+    belongs_to :user
+    validates :name, presence: true
```

Comment:
we add associations first then validations 
in priority such as  
belongs_to 
has_many
validations